benchmarks/benchmark_analysis.py

```diff
@@ -0,0 +1,213 @@
+            csv_file.write(f"{ligand_name}, {expt_DDG}, {expt_dDDG}\n")
+
+        # Calculated block
+        # print header for block
+        csv_file.write("# Calculated block\n")
+        csv_file.write("# Ligand1,Ligand2, calc_DDG, calc_dDDG(MBAR), calc_dDDG(additional)\n")
+        # Loop through simulation, extract ligand1 and ligand2 indices, convert to names, create string with
+        # ligand1, ligand2, calc_DDG, calc_dDDG(MBAR), calc_dDDG(additional)
+        # write string in csv file
+        for simulation in simulation_data:
+            out_dir = simulation.directory.split('/')[-1]
+            # getting integer indices
+            ligand1_id, ligand2_id = int(out_dir.split('_')[-1]), int(out_dir.split('_')[-2])  # CHECK ORDER!
+            # getting names of ligands
+            ligand1, ligand2 = lig_id_to_name[ligand1_id], lig_id_to_name[ligand2_id]
+            # getting calc_DDG in kcal/mol
+            calc_DDG = simulation.bindingdg.value_in_unit(unit.kilocalorie_per_mole)
+            # getting calc_dDDG in kcal/mol
+            calc_dDDG = simulation.bindingddg.value_in_unit(unit.kilocalorie_per_mole)
+            csv_file.write(
+                f"{ligand1}, {ligand2}, {calc_DDG}, {calc_dDDG}, 0.0\n")  # hardcoding additional error as 0.0
+
+
+# Defining command line arguments
+# fetching targets from github repo
+# TODO: This part should be done using plbenchmarks API - once there is a conda pkg
+targets_url = f"{base_repo_url}/raw/master/data/targets.yml"
+with urllib.request.urlopen(targets_url) as response:
+    targets_dict = yaml.safe_load(response.read())
+# get the possible choices from targets yaml file
+target_choices = targets_dict.keys()
+
+arg_parser = argparse.ArgumentParser(description='CLI tool for running perses protein-ligand benchmarks analysis.')
+arg_parser.add_argument(
+    "--target",
+    type=str,
+    help="Target biomolecule, use openff's plbenchmark names.",
+    choices=target_choices,
+    required=True
+)
+arg_parser.add_argument(
+    "--reversed",
+    action='store_true',
+    help="Analyze reversed edge simulations. Helpful for testing/consistency checks."
+)
+args = arg_parser.parse_args()
+target = args.target
+
+# Download experimental data
+# TODO: This part should be done using plbenchmarks API - once there is a conda pkg
+target_dir = targets_dict[target]['dir']
+ligands_url = f"{base_repo_url}/raw/master/data/{target_dir}/00_data/ligands.yml"
+with urllib.request.urlopen(ligands_url) as response:
+    ligands_dict = yaml.safe_load(response.read())
+
+# Get paths for simulation output directories
+out_dirs = get_simdir_list(is_reversed=args.reversed)
+
+# Generate list with simulation objects
+simulations = get_simulations_data(out_dirs)
+
+# Generate csv file
+csv_path = f'./{target}_arsenic.csv'
+to_arsenic_csv(ligands_dict, simulations, out_csv=csv_path)
+
+
+# TODO: Separate plotting in a different file
```

Comment:
@ijpulidos : I think we still want to separate the plotting into a different script!